### DISCUSSION TITLE
Is there a working group or plans for adding features specifically to support AVs?

### DISCUSSION THREAD

**Discussion by ajchatham** [openmobilityfoundation/mobility-data-specification#875] [Passenger Services]
I saw MDS called out in this SFMTA slide deck. 

We'd be willing to support this work with resources from the DIMO team, but it doesn't seem like it's being actively developed at the moment. https://www.sfmta.com/sites/default/files/reports-and-documents/2023/08/2023.08.07_cpuc_status_conference_8.7.2023_final.pdf

**Comment by schnuerle** [comment]
MDS can currently support AVs in the form of robotaxis and similar passenger services or fleets. You can see a blog post about it here as part of MDS 2.0. Many cities are actively working on implementing this now. 

https://www.openmobilityfoundation.org/data-digital-policy-how-mds-is-used-for-passenger-services/

**Comment by schnuerle** [comment]
FYI it's even better supported now in MDS 2.1.

https://github.com/openmobilityfoundation/governance/wiki/Release-2.1.0

https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.1.0/modes/passenger-services.md
